5205 TOGGLE Tom flyt

janitamarielle opened the pull request:
Implementerer tom flyt for kombinasjoner som ikke skal ha flyt. Lar SED-behandlingene gå til eksisterende sedbehandling og legger bare inn ny tomFlytMelding. Trekker VirksomhetMelding ut til felleskomponent. Toggler laging av url.

https://jira.adeo.no/browse/MELOSYS-5205

janitamarielle committed:
Implementerer tom flyt for kombinasjoner som ikke skal ha flyt. Lar SED-behandlingene gå til eksisterende sedbehandling og legger bare inn ny tomFlytMelding. Trekker VirksomhetMelding ut til felleskomponent. Toggler laging av url.

janitamarielle commented:
Merk at endreBehandlingModal sin lagUrl ikke er endret på. Tenkte dette burde gjøres etter https://github.com/navikt/melosys-web/pull/1558 er merget

martonSkjevelandNav commented on src/routing/url.ts:
```diff
@@ -33,6 +34,43 @@ export const lagUrl = (saksnummer: number | string, behandlingID: number, behand
 };
 
+export const lagUrlNy = (
+  // TODO  Når melosys.sakstema / melosys.behandle_alle_saker fjernes, kan denne funksjonen kalles lagUrl og den over lagUrlFraBehandlingstema
```
Er det ikke like greit å gjøre navneendringen nå, hvis de skal begge fortsatt brukes?

janitamarielle replied:
Fair point

martonSkjevelandNav commented on src/sider/tomFlyt/behandling.tsx:
```diff
@@ -0,0 +1,128 @@
+type PropsFromRedux = ConnectedProps<typeof connector>;
+
+interface MatchParams {
+  snr: string;
```
Antar snr er saksnummer? Ikke en forkortelse vi har brukt tidligere og dermed ikke helt intuitiv.
Bruk heller `saksnr`

martonSkjevelandNav replied:
Gjelder et par andre steder i PRen også

janitamarielle replied:
Endret overalt i koden 👍

janitamarielle committed:
Merge branch 'master' of https://github.com/navikt/melosys-web into MELOSYS-5205_tom-flyt

janitamarielle committed:
QA-kommentarer:
lagUrlNy -> lagUrl, lagUrl -> lagUrlFraBehandlingstema
toggle hvilken funksjon som kalles fra endreBehandlingModal.tsx nå som den er merget inn i master

janitamarielle committed:
snr -> saksnr
